js/main.js

```diff
@@ -0,0 +1,99 @@
+const sidebarElement = document.getElementById('sidebar');
+const addNewBookBtn = document.getElementById('new-book-btn');
+const closeSidebarBtn = document.getElementById('close-btn');
+const contentElement = document.getElementById('content');
+const formElement = document.getElementById('new-book-form');
+
+const library = JSON.parse(localStorage.getItem('library')) || [];
+
+function Book([title, author, descript, status = 'not-read']) {
+  this.title = title;
+  this.author = author;
+  this.descript = descript;
+  this.status = status;
+}
+
+function addBooks(book) {
+  library.push(book);
+}
+
```

Comment:
- lets add a means to remove the book as well

![Screenshot 2019-11-20 at 3 59 10 PM](https://user-images.githubusercontent.com/10922508/69249775-bc531d00-0bae-11ea-8f5d-0388ebbe33b8.png)